### PR TITLE
ZFIN-8936: Add option to hide emails for Lab and Person classes

### PR DESCRIPTION
**Pull request by rtaylorzfin**
The Lab and Person edit forms now offer the option to hide the email from potential scrapers.

### CODE REVIEW

File: source/org/zfin/profile/Organization.java [hunk 1]
```diff
@@ -328,33 +176,44 @@ public String getAbbreviationOrder() {
...
     @Override
     public String getEntityType() {
         return getType();
     }
 
+    public String getEmailIfVisible() {
+        if (emailPrivacyPreference == null) {
+            return email;
+        }
+        return emailPrivacyPreference.getEmailIfVisibleOrEmptyString(email, ProfileService::getCurrentSecurityUser);
```
Comment (cmpich) [inline source/org/zfin/profile/Organization.java:188]: It looks like this method is duplicated in Person, Lab and Organization. Could you write it once and re-use in all three entities?

File: source/org/zfin/profile/presentation/PersonController.java [hunk 2]
```diff
@@ -92,6 +92,9 @@ public String editView(@PathVariable String zdbID, Model model) {
         }
         model.addAttribute("showDeceasedCheckBox", showDeceasedCheckBox);
         model.addAttribute("countryList", profileService.getCountries());
+
+        model.addAttribute("emailPrivacyPreferenceList", profileRepository.getAllEmailPrivacyPreferences());
```
Comment (cmpich) [inline source/org/zfin/profile/presentation/PersonController.java:96]: Maybe, this duplicated code could also be put into one class and be re-used.

File: source/org/zfin/profile/Organization.java (hunk 1 above)
Comment (rtaylorzfin) [reply source/org/zfin/profile/Organization.java]: This is done with the latest commit

File: source/org/zfin/profile/presentation/PersonController.java (hunk 2 above)
Comment (rtaylorzfin) [reply source/org/zfin/profile/presentation/PersonController.java]: I updated it to use the profileService

File: source/org/zfin/db/postGmakePostloaddb/1152/db.changelog.master.xml [hunk 3]
```diff
@@ -6,5 +6,6 @@
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <include file="source/org/zfin/db/postGmakePostloaddb/1152/ZFIN-8922.sql" />
+    <include file="source/org/zfin/db/postGmakePostloaddb/1152/ZFIN-8936.sql" />
```
Comment (cmpich) [inline source/org/zfin/db/postGmakePostloaddb/1152/db.changelog.master.xml:9]: I assume this pertains to another ticket?

**Review by cmpich — APPROVED** [review]
Looks good.

File: source/org/zfin/db/postGmakePostloaddb/1152/db.changelog.master.xml (hunk 3 above)
Comment (rtaylorzfin) [reply source/org/zfin/db/postGmakePostloaddb/1152/db.changelog.master.xml:9]: It's for the same ticket. I forgot to commit it in the first PR.

**Pull request merged**